config/fam_client_interface_config.yaml

```diff
@@ -0,0 +1,42 @@
+#FAM Client Interface Service Config file
+
```

Comment:
Add an option to mention the client interface service IP address and port, which will be used during startup of the client interface service. Same can be passed through command line option while starting the service.